code/modules/surgery/organs/organ_external.dm

```diff
@@ -638,6 +633,23 @@ Note that amputating the affected organ does in fact remove the infection from t
 		owner.handle_splints()
 	return TRUE
 
+/obj/item/organ/external/proc/internal_bleed()
+	if(is_robotic())
+		return
+	if(HAS_TRAIT(owner, NO_BLOOD))
+		return
+	status |= ORGAN_INT_BLEEDING
+	owner.custom_pain("You feel something rip in your [name]!")
+
+/obj/item/organ/external/proc/fix_internal_bleeding()
+	if(is_robotic())
+		return FALSE
```

Comment:
I don't care enough to check, but can't a bodypart *become* robotic? 

You wouldn't want someone to be stuck with bleeding then.

Reply:
uhhh- through augmentation yes. *however* you cannot even start the surgery to fix IB on a robotic limb so- that probably could do with fixing.